src/argprocessing.cpp

```diff
@@ -841,7 +853,8 @@ process_arg(Context& ctx,
 
   // Same as above but options with concatenated argument beginning with a
   // slash.
-  if (args[i][0] == '-') {
+  if (args[i][0] == '-'
+      || (config.compiler_type() == CompilerType::cl && args[i][0] == '/')) {
     size_t slash_pos = args[i].find('/');
```

Comment:
Under Windows it doesn't work at all, because it could be ill formed like '-includeC:/path/to/header.h'.

Reply:
On the other hand, this is not correct even on POSIX, as e.g. '-include../inc/header.h' won't work either.

Ccache's handling of compiler options has always been best-effort, and there always will be things that won't be handled correctly just because nobody's been bothered to make ccache handle this or that case properly. It's just a question of where the limit is set. And given the situation, I think in this case it'd make sense to lower the bar, and fix it up more in follow-up commits. Otherwise there's a risk this ends up like #162 all over again.

Reply:
Several things are broken at this point. I have a more general solution to the problem on a branch, but I still need to prepare it for integration.
If @jrosdahl  is okay with that, I would also fix that in a later commit.